### PR TITLE
just fix some typo

### PR DESCRIPTION
**Pull request by jude-zhu**
fix some typo, Wilson will help me with the following proofreading.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -235,13 +235,13 @@ Math,1,3
 English,2,6
 ```
 
-You can specify `:IGNORE` to ignore the second column when importing data. In addition, all columns except the `:LABEL` column can be in any order. This way, for a large csv file, you can flexibly select the columns you need by setting the header.
+`:IGNORE` is to specify the row that you want to ignore when importing data. All columns except the `:LABEL` column can be in any order. Thus, for a large CSV file, you can flexibly select the columns you need by setting the header.
```
Comment (yixinglu) [inline README.md]: not row, but column

File: README.md [hunk 2]
```diff
@@ -9,24 +9,24 @@
 
 ## Introduction
 
-Nebula Importer is a CSV import tool for [Nebula Graph](https://github.com/vesoft-inc/nebula). It can read and import data in local CSV files.
+Nebula Importer is a CSV import tool for [Nebula Graph](https://github.com/vesoft-inc/nebula-docker-compose). It reads data from CSV files and inserts it into Nebula.
```
Comment (yixinglu) [inline README.md]: fix nebula graph repo link address

File: README.md [hunk 3]
```diff
@@ -236,13 +236,14 @@ Math,1,3
 English,2,6
 ```
 
-You can specify `:IGNORE` to ignore the second column when importing data. In addition, all columns except the `:LABEL` column can be in any order. This way, for a large csv file, you can flexibly select the columns you need by setting the header.
+
```
Comment (yixinglu) [inline README.md]: blank line

File: README.md [hunk 4]
```diff
@@ -257,15 +258,15 @@ In the preceding example, the source vertex of the edge is `:SRC_VID` (in column
 * `+` means inserting
 * `-` means deleting
 
-You can also specify label in edge CSV file header the same way with vertex.
+The same with vertex, you can specify label in edge CSV file header the same way with vertex.
```
Comment (yixinglu) [inline README.md]: "the same way with vertex" in the start and end of this line.

File: README.md [hunk 5]
```diff
@@ -279,7 +280,7 @@ $ go run importer.go --config /path/to/yaml/config/file
 
 ### From Docker
 
-With Docker you don't have to install golang locally. Pull Nebula Importer's [Docker Image](https://hub.docker.com/r/vesoft/nebula-importer) to import. The only thing to do is to mount the local configuration file and the CSV data files into the container as follows:
+With Docker, you don't have to install golang locally. Pull Nebula Importer's [Docker Image](https://hub.docker.com/r/vesoft/nebula-importer) to import. The only thing to do is to mount the local configuration file and the CSV data file into the container as follows:
```
Comment (yixinglu) [inline README.md]: maybe many data files

File: README.md [hunk 6]
```diff
@@ -9,24 +9,24 @@
...
 
-Before you try Nebula Importer, make sure you have:
+Before you start Nebula Importer, ensure:
 
-* Deployed Nebula Graph
-* Created schema for the space to import data
+* Nebula Graph is deployed
+* Schema is created
 
 Currently, there are three ways to deploy Nebula:
```
Comment (wilsonyou) [inline README.md]: It's better to use the full name: Nebula Graph

File: README.md [hunk 7]
```diff
@@ -9,24 +9,24 @@
...
 1. [nebula-docker-compose](https://github.com/vesoft-inc/nebula-docker-compose "nebula-docker-compose")
 2. [rpm Package](https://github.com/vesoft-inc/nebula/tree/master/docs/manual-EN/3.build-develop-and-administration/3.deploy-and-administrations/deployment)
 3. [from source](https://github.com/vesoft-inc/nebula/blob/master/docs/manual-EN/3.build-develop-and-administration/1.build/1.build-source-code.md)
 
-> The easiest way to try Nebula Graph is using [`docker-compose`](https://github.com/vesoft-inc/nebula-docker-compose).
+> The quickest way to deploy Nebula Graph is using [`docker-compose`](https://github.com/vesoft-inc/nebula-docker-compose).
 
 ## Prepare Configuration File
 
-Nebula-importer reads the CSV file to be imported and Nebula server data through the YAML configuration file. Here's an [example](example/example.yaml) of the configuration file and the CSV file. Detail descriptions for the configuration file see the following section.
```
Comment (wilsonyou) [inline README.md]: Nebula Importer reads the CSV files to be imported and data for Nebula Graph through the ....

File: README.md [hunk 8]
```diff
@@ -35,7 +35,7 @@ description: example
 
 * `version` is a **required** parameter that indicates the configure file's version, the default version is `v2rc1`.
```
Comment (wilsonyou) [inline README.md]: `version` is a **required** parameter that indicates the version number of the configuration file. The default version is `v2rc1`.

File: README.md [hunk 9]
```diff
@@ -35,7 +35,7 @@ description: example
...
 * `description` is an **optional** parameter that describes the configure file.
```
Comment (wilsonyou) [inline README.md]: configuration file

File: README.md [hunk 10]
```diff
@@ -35,7 +35,7 @@ description: example
...
-* `clientSettings` takes care all the Nebula related configurations.
```
Comment (wilsonyou) [inline README.md]: `clientSettings` takes care all the Nebula related configurations.

All configurations related to the **Nebula Graph** server are configured in the`clientSetting` field.

File: README.md [hunk 11]
```diff
@@ -49,8 +49,8 @@ clientSettings:
 ```
 
 * `clientSettings.concurrency` is an optional parameter that shows the concurrency of Nebula Graph Client, i.e. the connection number of Nebula Graph Server, the default value is 10.
```
Comment (wilsonyou) [inline README.md]: `clientSettings.concurrency` is an optional parameter that shows the concurrency of Nebula Graph Client, i.e. the connection number of Nebula Graph Server.  The default value is 10.

File: README.md [hunk 12]
```diff
@@ -49,8 +49,8 @@ clientSettings:
...
-* `clientSettings.channelBufferSize` is an optional parameter that shows the buffer size of cache queue for each Nebula Graph Client, the default value is 128.
```
Comment (wilsonyou) [inline README.md]: . The default value is 128

File: README.md [hunk 13]
```diff
@@ -49,8 +49,8 @@ clientSettings:
...
-* `clientSettings.space` is a **required** parameter that specifies which `space` the data will be importing into. Do not import data to multiple spaces at one time for the performance sake.
```
Comment (wilsonyou) [inline README.md]: `clientSettings.space` is a **required** parameter that specifies which `space` the data will be imported to. Do not import data to multiple spaces at one time for performance sake.

File: README.md [hunk 14]
```diff
@@ -74,17 +74,17 @@ files:
 
 ### CSV Data Files
 
-One CSV file can only store one type of vertex or edge. Vertices and edges of different schema should be stored in different files.
```
Comment (wilsonyou) [inline README.md]: One CSV file can only store one type of vertex or edge. Vertices and edges of different schemas must be stored in different files.

File: README.md [hunk 15]
```diff
@@ -74,17 +74,17 @@ files:
...
+One CSV file can only store one type of vertex or edge. Vertices and edges of the different schema should be stored in different files.
 
 * `path`: **Required**. Specifies the path where the CSV data file is stored. If a relative path is used, the path and directory of the current configuration file are spliced.
```
Comment (wilsonyou) [inline README.md]: I don't quite understand this part. It's a little tricky. Please check with developers.

File: README.md [hunk 16]
```diff
@@ -74,17 +74,17 @@ files:
...
 * `failDataPath`: **Required**. Specifies the file to insert the failed data output so that the error data is appended later.
 * `batchSize`: **Optional**. Specifies the batch size of the inserted data, the default value is 128.
```
Comment (wilsonyou) [inline README.md]: . The default value is 128

File: README.md [hunk 17]
```diff
@@ -74,17 +74,17 @@ files:
...
-* `type & csv`:  **Required**. Specifies the file type. Currently only CSV is supported. You can specify whether to include the header and the inserted and deleted labels in the CSV file.
+* `type & csv`:  **Required**. Specifies the file type. Currently, only CSV is supported. You can specify whether to include the header and the inserted and deleted labels in the CSV file.
   * `withHeader`: The default value is false, the format of the header is described below.
```
Comment (wilsonyou) [inline README.md]: `withHeader`: The default value is false. The format of the header is described below.

File: README.md [hunk 18]
```diff
@@ -74,17 +74,17 @@ files:
...
   * `withLabel`: The default value is false, the format of the label is described below.
```
Comment (wilsonyou) [inline README.md]: ditto

File: README.md [hunk 19]
```diff
@@ -74,17 +74,17 @@ files:
...
 
 * `schema`: **Required**. Describes the metadata information of the current data file. The schema.type has only two values: vertex and edge.
-  * When type is specified as vertex, details should be described in vertex field.
```
Comment (wilsonyou) [inline README.md]: When the type is specified as vertex, details must be described in the **vertex** field.

File: README.md [hunk 20]
```diff
@@ -74,17 +74,17 @@ files:
...
+  * When type is specified as vertex, details should be described in the vertex field.
   * When type is specified as edge, details should be described in edge field.
```
Comment (wilsonyou) [inline README.md]: ditto

File: README.md [hunk 21]
```diff
@@ -132,22 +132,22 @@ schema:
 `schema.edge` is a **required** parameter that describes the schema information of the inserted edge. Each edge contains the following three properties:
```
Comment (wilsonyou) [inline README.md]: `schema.edge` is a **required** parameter that describes the schema information of the inserted edges. Each edge contains the following three properties:

File: README.md [hunk 22]
```diff
@@ -132,22 +132,22 @@ schema:
...
 
 * `name`: The edge's name.
-* `withRanking`: Specifies the `rank` value of the given edge, used to tell different edges share the same edge type and vertices.
```
Comment (wilsonyou) [inline README.md]: `withRanking`: Specifies the `rank` value of the given edge. It is used to tell different edges sharing the same edge type and vertices.

File: README.md [hunk 23]
```diff
@@ -132,22 +132,22 @@ schema:
...
+* `withRanking`: Specifies the `rank` value of the given edge, used to tell different edges to share the same edge type and vertices.
 * `props`: Same as the above tag. Please be noted the property order here must be the same with that of the corresponding data in the CSV data file.
 
 Details of all the configurations please refer to [Configuration Reference](docs/configuration-reference.md).
```
Comment (wilsonyou) [inline README.md]: For details of all the configurations, refer to [Configuration Reference](docs/configuration-reference.md).

**Pull request merged**